app.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express")
+
+const {getTopicPath} = require('./controllers/controllers.topic')
+const {error404} = require('./controllers/controllers.errors')
+
+const app = express()
+app.use(express.json());
```

Comment:
can you remove this line up until the point where we need to use it to keep it strictly TDD

Reply:
just to confirm, you want to me to remove 4,5,6,7 ?

Reply:
just line 7 :)

Reply:
Done ✅